src/reducers/medication-reducers.js

```diff
@@ -323,6 +324,59 @@ const medicationReducers = (state = initialState, action) => {
         });
       }
 
+      case types.TAKE_SUGGESTION: {
+        if (action.suggestion && action.suggestion.actions) {
+          const { actions } = action.suggestion;
+          for (let i = 0; i < actions.length; i += 1) {
+            if (actions[i].type) {
+              if (!actions[i].description) {
+                console.error('Missing required "description" field in suggestion action', actions[i]);
+              }
+              if (actions[i].type === 'create' || actions[i].type === 'update') {
+                // Updating internal medication if new medication comes through suggestion
+                if (actions[i].resource && actions[i].resource.medicationCodeableConcept && state.fhirResource &&
+                  !isEqual(actions[i].resource.medicationCodeableConcept, state.fhirResource.medicationCodeableConcept)) {
+                  const newMedication = actions[i].resource.medicationCodeableConcept;
+                  if (newMedication.text && newMedication.coding && newMedication.coding[0] && newMedication.coding[0].code) {
```

Comment:
Do you need to log ```newMedication``` to the console to find out what it was missing if this check fails?